AEPEdgeIdentity.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = "AEPEdgeIdentity"
-  s.version          = "1.2.0"
```

Comment:
Also update the internal extension version and MARKETING_VERSION in the .xcodeproj